src/Hosting/SystemdServices/src/Microsoft.Extensions.Hosting.SystemdServices.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>.NET hosting infrastructure for Systemd Services.</Description>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
This is netstandard2.1 because of the `UnixDomainSocketEndPoint`. If desired, it can be made `netstandard2.0` by including an internal implementation.

Reply:
I don't think we have a compelling scenario for netstandard2.0 yet, so 2.1 is fine :)

Reply:
I agree for now maybe. We can polyfill if the demand is high.